fixtures/src/components/uncontrolled/date.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Date from 'carbon-react/lib/__experimental__/components/date';
+import { LogConsumer } from '../log';
+
+const UncontrolledDate = () => {
+  return (
+    <LogConsumer>
+      {(log) => {
+        const onChange = e => log(e, { method: 'onChange' });
+
+        const onBlur = e => log(e, { method: 'onBlur' });
+
+        return (
+          <React.Fragment>
+            <div id='uncontrolled_date'>
+              <h1>Uncontrolled Date</h1>
+              <ul>
+                <li>onChange handler should update the log when the value is changed, e.target.value should be the
+                  users input
```

Comment:
I think there should be an explanation about `e.target.value` containing `rawValue` and `formattedValue`